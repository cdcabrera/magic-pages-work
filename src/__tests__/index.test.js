const { magicPagesWork } = require('../index');

describe('Magic Pages Work', () => {
  it('should return pass-through values', () => {
    expect(magicPagesWork({ version: 'lorem-ipsum' })).toMatchSnapshot('Magic Pages Work');
  });
});
