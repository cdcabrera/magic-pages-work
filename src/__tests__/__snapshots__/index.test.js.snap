// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Magic Pages Work should return pass-through values: Magic Pages Work 1`] = `
{
  "version": "lorem-ipsum",
}
`;
